package.json

```diff
@@ -11,7 +11,7 @@
     "prepare": "npm run build",
     "build": "tsc --project ./tsconfig.json",
     "test": "BLUEBIRD_DEBUG=1 node --max_old_space_size=3072 node_modules/jasmine/bin/jasmine.js --stop-on-failure=true",
-    "lint": "eslint -c .eslintrc --max-warnings 0 src/**/*.ts",
+    "lint": "eslint -c .eslintrc --max-warnings 30 'src/**/*.ts'",
```

Comment:
Talked about out of band, new PR will fix these warnings. Please linky

Reply:
https://github.com/matrix-org/matrix-appservice-irc/pull/1289